src/client/components/VesselDetails.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from "react";
+import CardTeaser from "./CardTeaser";
+import { CardGroup, CardGroupHeader } from "./CardGroup";
+import VesselTeaser from "./VesselTeaser";
+
+function VesselDetailsComponent({
+  vesselName,
+  departureFrom,
```

Comment:
Would prefer `departsFrom` as it is more consistent with `arrivesAt`. Please rename all occurrences throughout the PR.

Reply:
renamed departureFrom to departsFrom